README.md

```diff
@@ -2,71 +2,65 @@
-* `main.go` contains the entry point code that starts the execution of the event source.
-* `Dockerfile` contains the directives necessary for building a container image for the event source.
-* `Makefile` contains goals for building, verifying and testing the code of the event source. Try `make help` to show all available goals for the event source.
+## Getting Started
 
-For detailed usage instructions about a particular source, please refer to its own `README.md` file.
+The following table lists the AWS services currently supported by TriggerMesh Sources for AWS and their support level.
 
-## Caveat
+|                            AWS Service                            |                  Documentation                   | Support Level |
+|-------------------------------------------------------------------|--------------------------------------------------|---------------|
+| [CodeCommit](https://aws.amazon.com/codecommit/)                  | [README](cmd/awscodecommitsource/README.md)      | alpha         |
+| [Cognito](https://aws.amazon.com/cognito/)                        | [README](cmd/awscognitoidentitysource/README.md) | alpha         |
+| [DynamoDB](https://aws.amazon.com/dynamodb/)                      | [README](cmd/awsdynamodbsource/README.md)        | alpha         |
+| [Kinesis](https://aws.amazon.com/kinesis/)                        | [README](cmd/awskinesissource/README.md)         | alpha         |
+| [Simple Notifications Service (SNS)](https://aws.amazon.com/sns/) | [README](cmd/awssnssource/README.md)             | alpha         |
+| [Simple Queue Service (SQS)](https://aws.amazon.com/sqs/)         | [README](cmd/awssqssource/README.md)             | alpha         |
 
-AWS events are very rich. AWS SNS and AWS CloudWatch can be used with almost every AWS service, hence there are many different ways to consume and/or receive AWS events. These sources represent one way of doing it.
+For detailed usage instructions about a particular source, please refer to the linked `README.md` files.
 
-## TriggerMesh Cloud Early Access
+## TriggerMesh Cloud
 
-These container sources can be used as is from this repo. You can also use them from our Cloud [https://cloud.triggermesh.io](https://cloud.triggermesh.io) where we have developed an enjoyable UI to configure them. Check out this snapshot:
+These container sources can be used as is from this repository or you can use our feature rich [TriggerMesh Cloud](https://cloud.triggermesh.io) offering where you can use our easy to use interface to configure them and also have access to other event source that are not publicly available.
 
-![TM cloud sources](./images/sources.png)
+![TriggerMesh Cloud](images/triggermesh-cloud.png)
 
 ## Roadmap
 
 * Add a more generic SNS source using an operator architecture
 * Add a CloudWatch source using an operator architecture
-* Use goroutines to make the sources more performant
+* Improve performance of the sources with the help of goroutines
```

Comment:
In hindsight I don't think that's something relevant for a roadmap, too many implementation details. Let's remove it or just say "Performance improvements".